packages/grid/_modules_/grid/locales/ukUA.ts

```diff
@@ -2,10 +2,36 @@ import { ukUA as ukUACore } from '@mui/material/locale';
 import { GridLocaleText } from '../models/api/gridLocaleTextApi';
 import { getGridLocalization, Localization } from '../utils/getGridLocalization';
 
+type PluralForm = {
+  single: string;
+  other: string;
+  many: string;
+};
+
+const getPluralForm = (
+  count: number,
+  options: PluralForm = {
+    single: 'активний фільтр',
+    other: 'активні фільтра',
+    many: 'активних фільтрів',
+  },
+) => {
```

Comment:
Way to go! :+1:
I see some room for improvement though:
- change `PluralForm` keys to `one`, `few` and `many` (inspired by 18next https://www.i18next.com/translation-function/plurals#languages-with-multiple-plurals)
- remove default `PluralForm` value, so that it's passed explicitly in each function call

Reply:
Yes, it works for me. Thanks